quickstarts/manage-data/airflow-airbyte-dbt.md

```diff
@@ -0,0 +1,243 @@
+cd airbyte_dbt_airflow_teradata
+```
+Open the code in your preferred IDE.
+
+
+## Setting up Teradata instance
+Follow the instructions from [Getting started with ClearScape Analytics Experience](https://developers.teradata.com/quickstarts/get-access-to-vantage/clearscape-analytics-experience/getting-started-with-csae/) and acquire following:
+
+*  Host
+* username
+* password
+
+Jot it down, it will be needed in next step.
+
+## Edit `profiles.yml` file
+Navigate to `dbt_project` folder, open `profiles.yml` file, it contains configuration for dbt to connect with your data platform. Update this file with your Teradata connection details:
+* Host (from previous step)
+* username (from previous step)
+* password (from previous step)
+* schema: `airbyte_td`
+
+And move the file in location shown below:
+
+<ProfileTabs/>
+
+## Airbyte setup
+
+1. After you install Airbyte OSS locally using `abctl`:
+
+```bash
+abctl local install
+```
+
+2. Generate and copy the credentials:
+
+```bash
+abctl local credentials
+```
+
+You can learn more about it from [here](https://docs.airbyte.com/using-airbyte/getting-started/oss-quickstart)
+
+3. Create a source
+    * Go to the Sources tab and click on `+ New source`.
+    * Search for “faker” using the search bar and select `Sample Data (Faker)`.
+    * Adjust the Count and optional fields as needed for your use case. You can also leave as is.
+    * Click on `Set up source`.
+
+4. Create a destination
+
+    * Go to the Destinations tab and click on `+ New destination`.
+    * Search for “teradata” using the search bar and select `Teradata Vantage`.
+    * Enter the connection details as needed.
+    * Click on `Set up destination`.
+
+5. Create a connection
+
+    * Go to the Connections tab and click on `+ New connection`.
+    * Select the source and destination you just created.
+    * Enter the connection details as needed.
+    * Click on `Set up connection`.
+
+![Airbyte Connection](../images/airbyte_connection.png)
+
+6.Airbyte `connection id`
+
+On the connection page,  from the URL, copy the value between `connections` and `status` 
+
+![Airbyte Connection id](../images/airbyte_connection_id.png)
+
+This will be needed later in the Airflow setup.
+
+## dbt setup
+
+The dbt project is already present in the `dbt_project` folder. 
+
+1. Navigate to the folder.
+
+```bash
+cd dbt_project
+```
+
+2. Create a Python virtual environment
+
+```bash
+python -m venv env
+```
+
+3. Activate the environment
+
+```bash
+source env/bin/activate
+```
+
+for Mac, Linux, or
+
+```bash
+env\Scripts\activate
+```
+
+for Windows
+
+4. Install `dbt-teradata` and `dbt-core` modules
+
+```bash
+pip install dbt-teradata
+```
+
+```bash
+pip install dbt-core>=1.8.0
+```
+
+5. Test the connection
+
+```bash
+dbt debug
+```
+
+If the output of above command is `All checks passed!` then we are good to move forward, else, check the values entered in `profiles.yml` and also make sure all the modules are installed correctly.
```

Comment:
This entire section is not needed as the new solution uses a Docker container with preinstalled libs.